oauth/OAuthUtility.php

```diff
@@ -146,8 +146,8 @@ public function apiCall($apiParams, $accessToken, $accessSecret, $method = "GET"
 
 	public function getIdentity($accessToken, $accessSecret)
 	{
-
-		global $toolUserAgent;
+		
+        global $toolUserAgent;
```

Comment:
Actually, @cyberpower678, since you don't make any other changes in this file, you can also just remove it from the pull request so the whitespace changes aren't submitted.